identity-and-access-management/secret-manager/how-to/create-secret.mdx

```diff
@@ -64,7 +71,14 @@ The [secret](/identity-and-access-management/secret-manager/concepts/#secret) cr
            The maximum file size for your secret is 64 KiB.
           </Message>
     7. Optionally, click <Icon name="toggle" /> to enable [secret protection](/identity-and-access-management/secret-manager/concepts/#secret-protection).
-    8. Click **Create secret**. The **Overview** tab of your secret displays.
+    8. Optionally, click <Icon name="toggle" /> next to **Single access** or **Time to Live** to apply an [ephemeral policy](/identity-and-access-management/secret-manager/concepts/#ephemeral-policy) to your secret and its versions.
```

Comment:
Not sure, but shouldn't it be "Time to live"?

Reply:
We decided to write it "Time to Live" in the wording because everyone writes it that way